compute/rn/release-information/release-notes-22-06-update2.adoc

```diff
@@ -0,0 +1,91 @@
+== 22.06 Update 2 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.xxx
+
+|Codename
+|Kepler, 22.06 Update 2
+//Tentative date
+|Release date
+|Sep 15, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, Fixes, and Performance Enhancements
+// GithubIssue PCSUP-issue#
+// GH#40449 PCSUP-10632
+* Fixed an issue that caused Defender to incorrectly report the Host OS as SLES15SP1 instead of SLES15.
+
+// GH#40673 #PCSUP-10988  #PCSUP-1940
+* Fixed an internal error that failed to refresh the vulnerability statistics under *Monitor > Vulnerabilities > Vulnerability Explorer*.
+
+// GH#39394	PCSUP-9241
+* Fixed an explicit deny error when sending a test alert to the AWS Security Hub integration. With this fix, the correct AWS product ARN for US and China regions are used.
+
+// GH#32310 PCSUP-10507
+* Fixed a bug in scanning images that mount on a directory when a parent directory is a symlink, causing many false binary detections.
+OR
+* Fixed a bug in scanning images on container defenders installed on CRI with CRI-O containerd that mount on a directory when a parent directory is a symlink, causing many false binary detections.
+OR
+* The image scans on container defenders installed on CRI with CRI-O containerd running from the host namespace are ignored to avoid false binary vulnerability detections.
+
+// GH#39262
+* Added support for more orchestrators:
+  
+** Google Kubernetes Engine (GKE) version 1.24.2 with containerd version 1.5.13
+
+** Elastic Kubernetes Service (EKS) version 1.23.7
+
+** Azure Kubernetes Service (AKS) version 1.24.3 running on Linux
+
+** AKS version 1.24.3 running on Windows
+
+** Lightweight Kubernetes (k3s) version v1.24.4+k3s1 with containerd v1.5.13-k3s1
+
+** Openshift version 4.11 with CRIO
+
+** Rancher Kubernetes Engine (RKE) version 1.24.4+rke2r1 with containerd 1.6.6-k3s1
+
+// GH# 39754
+* Prisma Cloud Compute now supports embedding a Defender to a CloudFormation Fargate task in the YAML format, in addition to the JSON format.
+
+** You can use a full CloudFormation template that contains other objects in addition to the Fargate task to generate a protected Fargate task definition.
+
+** You can use the Console (Manage > Defenders > Deployed Defenders) or the APIs (/api/22.06/defenders/fargate.yaml, /api/22.06/defenders/fargate.json) to complete the workflow.
```

Comment:
@ssugandh the path should be "Manage > Defenders > Deploy > Defenders" (the path you mentioned is the new path starting Lagrange)